.buildkite/pipeline.build.yml

```diff
@@ -163,6 +163,26 @@
     - if [ "${BUILDKITE_PULL_REQUEST}" = "false" ]; then python .buildkite/copy_files.py --destination docker_login; fi
     - python ./ci/build/build-docker-images.py --py-versions py310 --device-types cu111 cu112 cu113 cu116 --build-type BUILDKITE --build-base
 
+
+- label: ":docker: Build Images: py311 (1/2)"
```

Comment:
Let's remove the docker build jobs for now, this is not a blocker to generate wheels.